recipe/setupMvaMEt.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# These updates are reflected in two commit messages that are not yet merged
+# into working branches of CMSSW packages
+# cov patch https://github.com/rfriese/cmssw/commit/0fbc53683807893b8230d84e4cfb9d8381ab97c7
+# 76x patch https://github.com/rfriese/cmssw/commit/72cb039830ab9425b63eab4cb46b7d74674dce21
+
+echo "Setting up MVA MET for 76x by copying updated files from Tyler's 76x repo"
+
+for item in plugins/PFMETProducerMVA.cc plugins/PFMETProducerMVA.h python/mvaPFMET_cff.py src/PFMETAlgorithmMVA.cc test/mvaMETOnMiniAOD_cfg.py;
```

Comment:
i really don't like this. In the past, if we had to change something from a merge we used a patch (or you could create a tag in your personal cmssw repo and add here a git-merge).

See here: https://github.com/uwcms/FinalStateAnalysis/tree/miniAOD_dev_74X/recipe/patches